projects/models.py

```diff
@@ -1,88 +1,92 @@
 import json
 import logging
 
+from django.apps import apps
 from django.contrib.auth.models import Group, Permission
 from django.core.exceptions import ObjectDoesNotExist
 from django.db import models
 from django.db.models.signals import post_save
 from django.dispatch import receiver
+from django.utils.translation import gettext_lazy as _
 from guardian.shortcuts import assign_perm
 
 from access_management.permission_constants import (
     PROJECT_ADMIN_GROUP,
     manage_project_users_permission,
 )
 from access_management.utils import generate_groups_and_permission
-from catalogs.models import Catalog
 from components.componentio import EmptyComponent
-from components.models import Component
-from users.models import User
 
 logger = logging.getLogger(__name__)
 
 
 class Project(models.Model):
+    class ImpactLevel(models.TextChoices):
+        LOW = "low", _("Low")
+        MODERATE = "moderate", _("Moderate")
+        HIGH = "high", _("High")
+
+    class LocationChoices(models.TextChoices):
+        CMSAWS = "cms_aws", _("CMS AWS Commercial East-West")
+        GOVCLOUD = "govcloud", _("CMS AWS GovCloud")
+        AZURE = "azure", _("Microsoft Azure")
+        OTHER = "other", _("Other")
+
+    class StatusChoices(models.TextChoices):
+        ACTIVE = "active", _("Active")
+        ARCHIVED = "archived", _("Archived")
+
     title = models.CharField(
         max_length=100, help_text="Name of the project", unique=False
     )
     acronym = models.CharField(
         max_length=20, help_text="Acronym for the name of the project", unique=False
     )
     creator = models.ForeignKey(
-        User,
+        to="users.User",
         on_delete=models.PROTECT,
         default=None,
         related_name="projects_created",
         help_text="User id of the project creator",
     )
     components = models.ManyToManyField(
-        Component,
+        to="components.Component",
         blank=True,
         related_name="used_by_projects",
         help_text="Components that exist in the project",
     )
     catalog = models.ForeignKey(
-        Catalog,
+        to="catalogs.Catalog",
         null=False,
         default=None,
         on_delete=models.PROTECT,
         related_name="projects_catalog",
         help_text="Catalog id that this project uses",
     )
-    IMPACT_LEVEL_CHOICES = [
-        ("low", "Low"),
-        ("moderate", "Moderate"),
-        ("high", "High"),
-        ("pii/phi", "PII or PHI"),
-    ]
+    controls = models.ManyToManyField(
+        to="catalogs.Controls",
+        through="ProjectControl",
+        blank=True,
```

Comment:
I'm not sure `blank` makes sense here. Should be fine to exclude that option. If a project doesn't have any controls associated with it, the manager will just return an empty QuerySet